examples/multi-agent-debate/main.py

```diff
@@ -0,0 +1,504 @@
+import typer
+import json
+import logging
+from rich.prompt import Prompt, Confirm
+from typing import List, Tuple, Optional, Literal, Any
+
+import langroid as lr
+from langroid.language_models import OpenAIGPTConfig
+from langroid.agent.chat_agent import ChatAgent, ChatAgentConfig
+from langroid.agent.task import Task
+from langroid.agent.tools.metaphor_search_tool import MetaphorSearchTool
+from langroid.utils.logging import setup_logger
+from langroid.utils.configuration import set_global
+from langroid.agent.tools.orchestration import DoneTool
+
+from config import get_base_llm_config, get_global_settings
+from models import SystemMessages, Message
+
+DEFAULT_SYSTEM_MESSAGE_ADDITION = f"""
+            DO NOT REPEAT ARGUMENTS THAT HAVE BEEN PREVIOUSLY GENERATED 
+            AND CAN BE SEEN IN THE DEBATE HISTORY PROVIDED. 
+            """
+FEEDBACK_AGENT_SYSTEM_MESSAGE = f"""  
+            STEP 1: Provide Thorough Debate Feedback
```

Comment:
there's only 1 step, so no need to say STEP 1

Reply:
fixed